### DISCUSSION TITLE
Getting the RHS and the corresponding parameters

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1814] [Questions & Problems] [answered]
Forgive me if this information is in the documentation somewhere, I couldn't find it. I used to export simulations to Matlab to look at the RHS of particular compartments to figure out what was going on, it's my understanding that this is no longer possible. 

Is there a different way to examine the RHS equations and the parameters that comprise them? 

Thanks,

DS

**Comment by Yuri05** [comment, marked as answer]
Still doable if you start PK-Sim/MoBi in developer mode (s. bullet point 2 [here](https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Using-Matlab-Toolbox-with-PK-Sim-MoBi-Version-9.0-or-later#preparation))

**Reply by DonkeySaddle** [reply]
Great! Thanks @Yuri05!
